src/transformers/models/dac/configuration_dac.py

```diff
@@ -0,0 +1,49 @@
+"""Dac model configuration"""
+
+import numpy as np
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+class DacConfig(PretrainedConfig):
+    r"""
+    """
+
+    model_type = "dac"
+
+    def __init__(
+        self,
+        encoder_dim=64,
+        encoder_rates=[2, 4, 8, 8],
```

Comment:
Could we just have one argument:
```python
    upsampling_ratios=[2, 4, 8, 8],
```

And then set `downsampling_ratios = upsampling_ratios[::-1]` in the modelling code?